doc/jsk_tools/test_nodes/test_topic_published.rst

```diff
@@ -22,6 +22,22 @@ Parameters
 
   If it's ``True``, it tests if **not** published.
 
+- ``~condition_%d`` (``str``, Default: ``None``, Optional)
+
+  Check bool value condition using the given Python expression.
+  The Python expression can access any of the Python builtins plus:
+  ``topic`` (the topic of the message), ``m`` (the message) and ``t`` (time of message).
+
+  For example, topic is ``std_msgs/String`` and if you want to check whether a sentence is a ``hello``, you can do the following.
+
+  .. code-block:: bash
+    condition_0: m.data == 'hello'
```

Comment:
Could you add the information about handling `>`, `<` in xml, like https://docs.oracle.com/cd/A97335_02/apps.102/bc4j/developing_bc_projects/obcCustomXml.htm ?

Reply:
Thanks for your comments.
I added the document. Please check it.
https://github.com/jsk-ros-pkg/jsk_common/blob/cc9873caed9a27f2817e85000902c6244ea563a4/doc/jsk_tools/test_nodes/test_topic_published.rst